eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubmittedVariantRenormalizationProcessor.java

```diff
@@ -97,7 +97,7 @@ private boolean areContextAndLastNucleotideEqual(ISubmittedVariant variant) {
 
     private char getContextBaseInAssembly(ISubmittedVariant variant) {
         long contextPosition = variant.getStart() - 1;
-        String sequence = fastaSequenceReader.getSequence(variant.getContig(), contextPosition, contextPosition);
+        String sequence = fastaSequenceReader.getSequenceToUpperCase(variant.getContig(), contextPosition, contextPosition);
```

Comment:
This section shouldn't care about capitalization, so in order to avoid some string transformation I would recommend going back to the plain `getSequence` call.